src/libs/blueprint/conduit_blueprint_mesh_examples.cpp

```diff
@@ -3421,6 +3421,165 @@ adjset_uniform(Node &res)
     }
 }
 
+//-----------------------------------------------------------------------------
+void 
+polyhedral_chain(index_t howmany, // how long the chain ought to be
+                 Node &chain)
+{
+    Node &chain_coords = chain["coordsets/coords"];
+    Node &chain_topo = chain["topologies/topo"];
+    Node &chain_fields = chain["fields"];
+
+    chain_coords["type"] = "explicit";
+
+    // Each cube requires 8 vertices, and each triangular prism requires 6
+    // There are two prisms for every cube, so 8+6+6 = 20 vertices
+    chain_coords["values/x"].set(conduit::DataType::int64(howmany * 20));
```

Comment:
The comment here is great for context, but I think using a variable name to explain what the number `20` means would be even better! For example:

```
const index_t num_verts_per_hex = 6;
const index_t num_verts_per_triprism = 8;
const index_t num_verts_per_chainpair = num_verts_per_hex + 2 * num_verts_per_triprism;

chain_coords["values/x"].set(conduit::DataType::int64(howmany * num_verts_per_chainpair));
```